tsconfig.production.json

```diff
@@ -0,0 +1,7 @@
+{
+  "extends": "./tsconfig.json",
+  "exclude": [
+    "node_modules",
+    "cypress"
```

Comment:
awkward just for this line due to eslint

Reply:
yeah, I tried many options but it breaks due to eslint dependency. Might need to optimize soon but so far this works.